typescript/cli/src/commands/options.ts

```diff
@@ -164,6 +164,20 @@ export const validatorCommandOption: Options = {
   demandOption: true,
 };
 
+export const transactionsCommandOption: Options = {
+  type: 'string',
+  description: 'The transaction input file path.',
+  alias: ['t', 'txs', 'txns'],
```

Comment:
I've noticed that when commands have aliases that aren't one letter, the yargs help print looks weird. Is that the case here?

Reply:
<img width="592" alt="Screenshot 2024-07-09 at 10 14 32 PM" src="https://github.com/hyperlane-xyz/hyperlane-monorepo/assets/15343884/c56595fe-0cc7-4f68-9103-7dc9cfea4fe1">

hmm looks fine to me